src/erlfmt.erl

```diff
@@ -463,9 +470,22 @@ read_nodes(
                 Warnings
             )
     end;
-read_nodes(Other, FileName, _Pragma, NodeAcc, Warnings, _TextAcc) ->
+read_nodes(Other, FileName, _PragmaFlag, NodeAcc, Warnings, _TextAcc) ->
     read_nodes_loop(Other, FileName, NodeAcc, Warnings).
 
+skip_nodes(
+    Cont,
+    FileName,
+    TextAcc
+) ->
```

Comment:
This could be probably folded back to a single line
```suggestion
skip_nodes(Cont, FileName, TextAcc) ->
```

Reply:
Good idea